CHANGELOG.md

```diff
@@ -13,7 +15,7 @@
   * `k8s.io/client-go` from `v0.30.0` to `v0.30.3`
   * `sigs.k8s.io/controller-runtime` from `v0.18.1` to `v0.18.5`
 * [ENHANCEMENT] Update Go to `1.23`. #168
-* [ENHANCEMENT] When mirroring replicas of statefulset, rollout-operator can now skip writing back number of replicas to reference resource, by setting `grafana.com/rollout-mirror-replicas-from-resource-write-back-status-replicas` annotation to `false`. #169
+* [ENHANCEMENT] When mirroring replicas of statefulset, rollout-operator can now skip writing back number of replicas to reference resource, by setting `grafana.com/rollout-mirror-replicas-from-resource-write-back` annotation to `false`. #169
```

Comment:
changing the changelog - are you doing 0.19.1 with it?

Reply:
Fixed in https://github.com/grafana/rollout-operator/pull/171/commits/f2944e30999c361e4e9ac94f22079bef42de9432.